### PR TITLE
update: pwn

### PR DESCRIPTION
**Pull request by Vojtch159**
Hi, I've updated metadata for PWN. Let me know if two paragraphs in `description` are valid, thanks! I also changed the category to lending as we're not only NFT focused.

### CODE REVIEW

File: defi/src/protocols/data3.ts [hunk 1]
```diff
@@ -15302,20 +15302,20 @@ const data3: Protocol[] = [
...
-    description: "PWN is a peer-to-peer lending protocol designed for long-term mortgage-like loans of digital assets, backed by other digital assets. Borrowers can receive a fixed-duration loan of fungible tokens backed by fungible or non-fungible tokens, while lenders can earn interest by granting these loans. The protocol is trustless, immutable and operates without the need for oracles or protocol-managed liquidations.",
-    chain: "Polygon",
+    description: "PWN is a peer-to-peer permissionless lending protocol. In the true spirit of DeFi composability, PWN users can borrow against—or lend using—any asset in their wallet (ERC-20, NFT, and even a bundle of both), while enjoying absolute flexibility on the loan terms (LTV, duration, APY, etc.). The oracle-less nature of the protocol protects all PWN loans from price-based liquidations. The only way to lose collateral is to default on a loan. \nOperating on seven EVM-compatible networks, PWN opens up new liquidity avenues, composability options, and leverage opportunities, while giving its users both optimized capital efficiency and predictability on both sides of the loan. A win on all fronts.",
+    chain: "Ethereum",
```
Comment (Define101) [inline defi/src/protocols/data3.ts:15306]: Please keep Ethereum here

Comment (Vojtch159) [reply defi/src/protocols/data3.ts:15306]: @Define101 Apologies, not sure I understand what should I change. The proposed change is from `Polygon` to `Ethereum`. Did you ment to keep `Polygon`?

Comment (Define101) [reply defi/src/protocols/data3.ts:15306]: My bad. I am sorry, i wanted to say keep Polygon
So the chain field is meant for the first blockchain where the project launch, even if they are not active there anymore

Comment (Vojtch159) [reply defi/src/protocols/data3.ts:15306]: alright, thanks, changed!

**Pull request merged**